grabbit/src/test/groovy/com/twcable/grabbit/jcr/ProtoNodeDecoratorSpec.groovy

```diff
@@ -0,0 +1,127 @@
+package com.twcable.grabbit.jcr
+
+import com.twcable.grabbit.proto.NodeProtos
+import com.twcable.grabbit.proto.NodeProtos.Node as ProtoNode
+import spock.lang.Specification
+
+import javax.jcr.Node
+import javax.jcr.Session
+
+import static javax.jcr.PropertyType.STRING
+import static org.apache.jackrabbit.JcrConstants.JCR_MIXINTYPES
+import static org.apache.jackrabbit.JcrConstants.JCR_PRIMARYTYPE
+
+@SuppressWarnings("GroovyAccessibility")
+class ProtoNodeDecoratorSpec extends Specification {
+
+    ProtoNode decoratedProtoNode
+
+
+    def setup() {
```

Comment:
I believe it is a good candidate for `@shared` because it does add to more unnecessary setup but i am fine otherwise too as the results remains the same.